src/Pages/Expenses.cshtml

```diff
@@ -139,7 +139,6 @@ else
                                         </div>
                                         <div class="col-12 col-md-5 col-lg-4">
                                             <span class="text-info">Amount:</span>
-                                            @*@NumberFormatter.FormatCurrency(item.Amount, Model.CurrencyException? item.Currency : Model.CurrencySymbol)*@
                                             @NumberFormatter.FormatCurrency(item.Amount, Model.CurrencySymbol)
```

Comment:
So we don't need to use that commented line? I remember you've put it here to use later?

Reply:
I completely forgot about this :D I'll go and actually check out how this works and see if we need to keep this or not